e2e/config.js

```diff
@@ -48,6 +48,11 @@ module.exports = {
     email: 'hmcts.civil+organisation.1.superuser@gmail.com',
     type: 'systemupdate'
   },
+  judgeUser: {
+    password: defaultPassword,
+    email: 'judge-civil-02@example.com',
```

Comment:
Judge user should be used from ref data users. I do not see judge-civil-02@example.com user on AAT. so the tests never pass